src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/activation.cpp

```diff
@@ -222,6 +223,7 @@ std::string ActivationLayerCPUTest::getPrimitiveType(const utils::ActivationType
        (activation_type == utils::ActivationTypes::Negative) ||
        (activation_type == utils::ActivationTypes::IsNaN) ||
        (activation_type == utils::ActivationTypes::IsFinite) ||
+       (activation_type == utils::ActivationTypes::HSigmoid) ||
```

Comment:
The [ARM32 tests](https://github.com/openvinotoolkit/openvino/actions/runs/15702708778?pr=31010) are failed with the following error:
```
primType is unexpected : acl_f32 Expected : ref_f32
```

Since on ARM32 platforms `HSigmoid` is decomposed into simple operations which are executed using ACL Executor, execution primitive type is `acl`. So could you please remove this line? Then expected primitive type will be `acl` as needed

Reply:
Oh, I understood my mistake, and also fixed it in https://github.com/openvinotoolkit/openvino/pull/31010/commits/1061a4eb04659e15a844cd266324f0d4bcadc1e4